app/models/link.rb

```diff
@@ -1,6 +1,7 @@
 class Link < ApplicationRecord
   has_many :checks
   has_many :monitor_links
```

Comment:
Does this still need to be a `has_many` following this change?

Reply:
Yep, its a many to many relationship :)